lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -416,7 +416,18 @@ sub serial_console_diag_banner {
     # max_length - length of the text - 4x2 dividing spaces
     my $symbol_fill = ($max_length - length($input_text) - 8) / 2;
     $input_text = '#' x $symbol_fill . ' ' x 4 . $input_text . ' ' x 4 . '#' x $symbol_fill;
-    script_run($input_text, quiet => 1, die_on_timeout => 0, timeout => 1);
+    my $fill_line = '#' x length($input_text);
+
+    # This will print banner in serial terminal:
```

Comment:
what is this comment about? Is this just a Perl comment?

Reply:
Yep just a perl comment to show an example of what will the function do. 
However, good you pointed out,  I will move it to function description as it seems like a better place for it.